lib/timeline/component/ItemSet.js

```diff
@@ -151,9 +151,9 @@ function ItemSet(body, options) {
   this.groupTouchParams = {};
   // create the HTML DOM
 
-  this._create();
-
   this.setOptions(options);
```

Comment:
Ummm I think you read it wrong. `_create` is after `_setOptions`. originally it was first create and then setOptions. I didn't understand why but I'm going to return it as it was before...